### PR TITLE
doc: gas cost parameter chapter

### PR DESCRIPTION
**Pull request by jakmeier**

### CODE REVIEW

**Comment by jakmeier** [comment]
So far, this is just a high level overview and some TODOs. I'd like to merge this in if the general structure of it makes sense. Then I will fill in gaps later, step by step as I find time for it.

**Review by akhi3030 — APPROVED** [review]
Approving to unblock the merge.  I'll do a more detailed review in a couple of hours.  Any feedback can be incorporated into future PRs.

**Review by akhi3030 — COMMENTED** [review]
I am also happy with the proposed structure here.

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -12,6 +12,10 @@
   - [Transaction Routing](./architecture/how/tx_routing.md)
 - [Trie](./architecture/trie.md)
 - [Network](./architecture/network.md)
+- [Gas Cost Parameters](./architecture/gas_params/README.md)
```
Comment (matklad) [inline docs/SUMMARY.md:15]: Paths here are visible in urls, so I think it's marginally better to stick to short, catchy names. In this case, `gas` would be a bit better than `gas_parms`.

File: docs/architecture/gas_params/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Gas Cost Parameters
+
+NEAR charges gas when executing users' WASM code. The how and why is described
```
Comment (matklad) [inline docs/architecture/gas_params/README.md:3]: gas is charged even without wasm (transfers and such).

**Pull request merged**